qemu/tests/cfg/qsd_blockdev_check.cfg

```diff
@@ -0,0 +1,23 @@
+- qsd_blockdev_check:
+    type = qsd_blockdev_check
+    start_vm = no
+    images = ""
+    # QSD attributes
+    qsd_namespaces = "qsd1"
+    qsd_images_qsd1 = "stg1 stg2"
+    qsd_force_create_qsd1 = yes
+    # Image attributes
+    image_name_stg1 = images/stg1
+    image_size_stg1 = 128M
+    image_name_stg2 = images/stg2
+    image_size_stg2 = 256M
+
+    qsd_cmd_lines += " --object iothread,id=iothread1;"
+    # Image throttle and export attributes
+
+    qsd_image_protocol_stg1 = {"aio":"native","cache":{"direct":true,"no-flush":false}}
+    qsd_image_format_stg1 = {"driver":"qcow2","read-only":true,"detect-zeroes":"unmap","discard":"unmap","force-share":true}
+    qsd_image_protocol_stg2 = {"aio":"threads","cache":{"direct":false,"no-flush":true}}
+    qsd_image_format_stg2 = {"driver":"raw","read-only":false,"detect-zeroes":"on","discard":"ignore","force-share":false}
```

Comment:
Suggest splitting it into multiple lines.

Reply:
updated